templates/web/pages/user_form/user_form.tt.js

```diff
@@ -1,15 +1,15 @@
 \$('#pro').change(function() {
 	if (\$(this).prop('checked')) {
 		\$('.pro_org_display').show();
-		\$('.tr_teams').hide();
+		\$('#teams_section').hide();
 	} else {
 		\$('.pro_org_display').hide();
-		\$('.tr_teams').show();
+		\$('#teams_section').show();
 	}
 	\$(document).foundation('equalizer', 'reflow');
 });
 
 if (\$('#pro').prop('checked')) {
 	\$('.pro_org_display').show();
-	\$('.tr_teams').hide();
+	\$('#teams_section').hide();
```

Comment:
Here we have an id: $('#teams_section') and in user_form_page.tt.html we have a class:
```
    .teams_section {
        display: block;
    }
```
I believe one of those is probably wrong, isn't it.

Reply:
Good catch. The style is not needed anymore, I'll remove it.

Reply:
fixed by https://github.com/openfoodfacts/openfoodfacts-server/pull/5287